tests/style_test.rs

```diff
@@ -0,0 +1,54 @@
+extern crate yoga;
+
+use yoga::{Direction, FlexDirection, Node, StyleUnit, Undefined};
+
+#[test]
+fn test_copy_style_same() {
+	let node0 = Node::new();
+	let node1 = Node::new();
+	assert!(!node0.is_dirty());
+
+	node0.copy_style(&node1);
+	assert!(!node0.is_dirty());
+}
+
+#[test]
+fn test_copy_style_modified() {
+	let node0 = Node::new();
+	assert!(!node0.is_dirty());
+	assert_eq!(FlexDirection::Column, node0.get_flex_direction());
+
+	match node0.get_style_max_height() {
```

Comment:
Could this simply be `assert_eq!(StyleUnit::UndefinedValue, node0.get_style_max_height());` ?

Reply:
Will change.